packages/starlight/utils/createTranslationSystem.ts

```diff
@@ -72,12 +72,12 @@ function buildDictionary(
 	base: (typeof builtinTranslations)[string],
 	...dictionaries: (i18nSchemaOutput | undefined)[]
 ) {
-	const dictionary = { ...base };
+	const dictionary: typeof base = { ...base };
```

Comment:
This is implicit and in my testing it looks like we can do without:
```suggestion
	const dictionary = { ...base };
```